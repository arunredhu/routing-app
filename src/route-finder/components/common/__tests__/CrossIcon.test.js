import React from 'react';
import renderer from 'react-test-renderer';
import CrossComponent from '../crossIcon';


describe('CrossComponent Test', () => {
    it('render correctly', () => {
        const inputType = "start";
        const resetInputDetails = jest.fn()

        const tree = renderer.create(<CrossComponent inputType={inputType}
            resetInputDetails={resetInputDetails} />).toJSON();

        expect(tree).toMatchSnapshot();
    });
});