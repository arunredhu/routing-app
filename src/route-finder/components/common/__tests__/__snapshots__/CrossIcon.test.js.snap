// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CrossComponent Test render correctly 1`] = `
<button
  onClick={[Function]}
  style={
    Object {
      "margin": "5px",
    }
  }
>
  <i
    className="fa fa-times"
  />
  X
</button>
`;
